browserify-test/script.js

```diff
@@ -0,0 +1,16 @@
+require('../source-map-support').install();
+
+var foo = function () {
+    throw new Error('foo');
+};
+
+try {
+    foo();
+} catch (error) {
+    if (/\bscript\.js\b/.test(error.stack)) {
+        document.body.appendChild(document.createTextNode('Test passed'));
+    } else {
+        document.body.appendChild(document.createTextNode('Test failed'));
+        console.log(error.stack);
+    }
+}
```

Comment:
Fixed test,
For unknown (at least for me) reasons browserify lost source map generated by coffee, then I decide to use only JS file which works fine

Reply:
I think that the test was there to ensure that this library works together with CoffeeScript, I don't think that we should remove it 🤔

Reply:
Are you sure about that? All tests are run on Travis before we merge anything 🤔